pymongo/collection.py

```diff
@@ -159,13 +158,47 @@ def __init__(
           - `session` (optional): a
             :class:`~pymongo.client_session.ClientSession` that is used with
             the create collection command
-          - `encrypted_fields`: **(BETA)** Document that describes the encrypted fields for
-            Queryable Encryption. If provided it will be passed to the create collection command.
           - `**kwargs` (optional): additional keyword arguments will
             be passed as options for the create collection command
 
+        All optional parameters should be passed as keyword arguments to this method. Valid
+        options include, but are not limited to::
+
+          - ``encryptedFields`` (dict): **(BETA)** Document that describes the encrypted fields for
+            Queryable Encryption. For example::
+
+                {
+                  "escCollection": "enxcol_.encryptedCollection.esc",
+                  "eccCollection": "enxcol_.encryptedCollection.ecc",
+                  "ecocCollection": "enxcol_.encryptedCollection.ecoc",
+                  "fields": [
+                      {
+                          "path": "firstName",
+                          "keyId": Binary.from_uuid(UUID('00000000-0000-0000-0000-000000000000')),
+                          "bsonType": "string",
+                          "queries": {"queryType": "equality"}
+                      },
+                      {
+                          "path": "ssn",
+                          "keyId": Binary.from_uuid(UUID('04104104-1041-0410-4104-104104104104')),
+                          "bsonType": "string"
+                      }
+                    ]
+                }
+
+          - ``clusteredIndex`` (dict): Document that specifies the clustered index
+            configuration. It must have the following form::
+
+                {
+                    // key pattern must be {_id: 1}
+                    key: <key pattern>, // required
+                    unique: <bool>, // required, must be ‘true’
+                    name: <string>, // optional, otherwise automatically generated
+                    v: <int>, // optional, must be ‘2’ if provided
+                }
+
```

Comment:
Let's remove this section to avoid duplicating the docs. We already document encryptedFields/clusteredIndex and other options in create_collection and instantiating a Collection directly is uncommon.

Reply:
Done.